src/main/webapp/js/common.js

```diff
@@ -179,6 +181,19 @@ $(document).on('ajaxComplete ready', function() {
 });
 
 /**
+ * @param elements:
+ * identifier that points to elements with
+ * class: profile-pic-icon-click or profile-pic-icon-hover
+ */
```

Comment:
Follow the normal javadoc comment format.
Do not mention where the function is used. A function should be independent of the call site.